pyproject.toml

```
@@ -0,0 +1,28 @@
+[build-system]
+requires = ['setuptools >= 61.0']
+build-backend = 'setuptools.build_meta'
+
+[project]
+name = 'occameracontrol'
+version = '0.1.0'
+authors = [
+    {name = 'Lars Kiesow', email = 'lkiesow@uos.de'},
+    {name = 'Jan-Matthis Niermann', email = 'janiermann@uos.de'}
+]
+description = 'Automated Camera Control for Opencast'
+readme = {file = 'README.md', content-type = 'text/markdown'}
+requires-python = ">=3.9"
+license = {file = 'LICENSE'}
+dependencies = [
+    'confygure>=0.1.0',
+    'prometheus-client',
+    'python-dateutil',
+    'requests'
+]
+
+[project.urls]
+Homepage = 'https://github.com/virtUOS/opencast-camera-control'
+Repository = 'https://github.com/virtUOS/opencast-camera-control'
+
+[project.scripts]
+occameracontrol = 'occameracontrol.__main__:main'
```

Same here. Fewer restrictions apply and I would suggest using the name `opencast-camera-control`.

```suggestion
opencast-camera-control = 'occameracontrol.__main__:main'
```